test/e2e/app-dir/metadata-dynamic-routes/app/sitemap-video/sitemap.ts

```diff
@@ -0,0 +1,23 @@
+import { MetadataRoute } from 'next'
+
+export default function sitemap(): MetadataRoute.Sitemap {
+  return [
+    {
+      url: 'https://example.com',
+      lastModified: '2024-01-01',
+      changeFrequency: 'daily',
+      priority: 0.5,
+    },
+    {
+      url: 'https://example.com/about',
+      lastModified: '2024-01-01',
+      videos: [
+        {
```

Comment:
Let's also test all the other properties that defined in the type

Reply:
added the all props in test case @huozhi